pex/resolver.py

```diff
@@ -718,24 +730,8 @@ def add_installation(install_result):
                 if distribution_satisfies_requirement(distribution, req)
                 and installed_distribution.target.requirement_applies(req)
             ]
-            if len(direct_reqs) > 1:
```

Comment:
In case the logic of eliminating this nannying isn't clear here: the resolver (Pip) has solved the requirements successfully at this point. That means if there are dup requirements, one of two situations is in play:
1. The requirements were compatible. An example is ["PyJWT", "PyJWT==1.7.1"].
2. The requirements were incompatible but only 1 was in-play, so ... they were compatible - everything taken into account: ["PyJWT==1.7.1; python_version == '2.7.*'", "PyJWT>1.7.1; python_version >= '3.6'"]

As such, there is no need to second guess. In case 1, where there are 2 requirements, the runtime `sys.path` assembly logic will necessarily find the one PyJWT distribution that was resolved satisfies both requirements. In case 2, the runtime `sys.path` assembly logic will find only one of the two requirements is active per `Marker.evaluate()`.